assets/js/components/user-input/UserInputPreview.js

```diff
@@ -79,80 +58,90 @@ export default function UserInputPreview( { noFooter, back, goTo, redirectURL }
 		USER_INPUT_ANSWERS_ROLE,
 	} = getUserInputAnwsers();
 
+	const { error } = useSelect( ( select ) => ( {
+		error: select( CORE_USER ).getErrorForAction( 'saveUserInputSettings', [] ),
+	} ) );
+
+	const [ single, setSingle ] = useQueryArg( 'single', false );
+
+	const editSingle = useCallback(
+		( questionNumber ) => {
+			setSingle( 'user-input' );
+			goTo( questionNumber );
+		},
+		[ single ],
+	);
```

Comment:
The `single` value isn't needed here. We can simplify it to be:

```suggestion
	const [ , setSingle ] = useQueryArg( 'single', false );

	const editSingle = useCallback( ( questionNumber ) => {
		setSingle( 'user-input' );
		goTo( questionNumber );
	}, [] );
```

Reply:
I wasn't able to do this because the `useQueryArgs` hook doesn't update in the parent component when set in the child component. We'd need to work on the hook some more to get this to work as expected.